### DISCUSSION TITLE
User does not see logo if there are no read asset permissions set

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#5277] [Error / Bug Report] [answered]
Hi,

Well this seems kind ofa strange issue.

If user has read:assets permission and read:assets page rule, then he sees the Site logo.

Image with permissions:
![image](https://user-images.githubusercontent.com/5992169/167624042-c2649f24-e18a-46e1-8a2f-d7eaadaf3cb5.png)

Image without permissions:
![image](https://user-images.githubusercontent.com/5992169/167624140-e13dc3cf-3e53-436b-b453-df23a40ac337.png)

To be fair, I don't think the Site logo should be tied to asset permissions.

**Comment by NGPixel** [comment, marked as answer]
This is being addressed in v3.

**Reply by ghost** [reply]
Thank you.
